src/cpu-info.spec.ts

```diff
@@ -1,20 +1,18 @@
-import { getCpuInfo } from './cpu-info';
+import { CpuInfo } from './cpu-info';
 
-describe('getCpuInfo', () => {
+describe('CpuInfo', () => {
   it('should get correct CPU information when available', () => {
     expect.assertions(1);
 
-    jest.spyOn(Navigator.prototype, 'hardwareConcurrency', 'get').mockReturnValue(1);
+    jest.spyOn(CpuInfo, 'getNumLogicalCores').mockReturnValue(1);
```

Comment:
Let's keep it so that these tests still get the value of `navigator.hardwareConcurrency`. There's not much use in mocking a function like this -- it's essentially saying, "assuming this function always returns 1, does this function return 1?" which isn't very useful.